include/ringmesh/mesh/mesh.h

```diff
@@ -52,15 +52,13 @@ namespace GEO
 
 namespace RINGMesh
 {
-    FORWARD_DECLARATION_DIMENSION_CLASS( GeoModel );
     FORWARD_DECLARATION_DIMENSION_CLASS( MeshBaseBuilder );
     FORWARD_DECLARATION_DIMENSION_CLASS( PointSetMeshBuilder );
     FORWARD_DECLARATION_DIMENSION_CLASS( LineMeshBuilder );
     FORWARD_DECLARATION_DIMENSION_CLASS( SurfaceMeshBuilder );
     FORWARD_DECLARATION_DIMENSION_CLASS( VolumeMeshBuilder );
     FORWARD_DECLARATION_DIMENSION_CLASS( SurfaceMesh );
 
-    struct EdgeLocalVertex;
     struct ElementLocalVertex;
     struct PolygonLocalEdge;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant forward declaration; the declaration was already made. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3231)